packages/monaco-editor-wrapper/src/editorAppBase.ts

```diff
@@ -167,25 +185,11 @@ export abstract class EditorAppBase {
 
     private updateAppConfig(modelUpdate: ModelUpdate) {
         const config = this.getConfig();
-        if (modelUpdate.code !== undefined) {
-            config.code = modelUpdate.code;
-        }
-
-        if (modelUpdate.languageId !== undefined) {
-            config.languageId = modelUpdate.languageId;
-        }
-
-        if (modelUpdate.uri !== undefined) {
-            config.codeUri = modelUpdate.uri;
-        }
-
-        if (modelUpdate.codeOriginal !== undefined) {
-            config.codeOriginal = modelUpdate.codeOriginal;
-        }
-
-        if (modelUpdate.codeOriginalUri !== undefined) {
-            config.codeOriginalUri = modelUpdate.codeOriginalUri;
-        }
+        config.languageId = modelUpdate.languageId;
```

Comment:
No undefined checks needed anymore?

Reply:
Because the change from undefined to not undefined is already a change.